src/Components/CartItems.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+
+function CartItems(){
+    return(  <div className="ItemsInCart"
+    style={{ display: "flex", flexDirection: "column", gap : "1rem" }}
```

Comment:
maybe you can avoid using inline styling

Reply:
yes I'll make it a class